### PR TITLE
feat(function/exports): AddSharedInventory(society)

### PR DESCRIPTION
**Pull request by wasabirobby**
Gives the ability to create shared addon inventories during runtime. Also exports added for easy integration into ESX add-ons in the future.

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -92,6 +92,19 @@ function GetSharedInventory(name)
 	return SharedInventories[name]
 end
 
+function AddSharedInventory(society)
```
Comment (Gellipapa) [inline server/main.lua:95]: Please handle society is nil or not a string.

**Comment by Gellipapa** [comment]
@wasabirobby Hi! Please add paramters handling. Thank you.

**Review by wasabirobby — COMMENTED** [review]
All done

**Comment by Gellipapa** [comment]
@wasabirobby Thank you i merged.

**Pull request merged**